### PR TITLE
Small updates in docs files and structure

### PR DESCRIPTION
**Pull request by rstens**
Set of smaller updates to add to the test directory structure and documentation.

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-vmvfjv-tools.pathfinder.gov.bc.ca/dashboard?id=common-forms-toolkit-pr-123

File: tests/functional-tests/readme.md [hunk 1]
```diff
@@ -6,22 +6,25 @@ This is an example of using Browserstack in conjunction with Geb/Spock tests as
...
- 
+
  It works equally well on Windows, Mac, and Linux hosts since tests run remotely, and it works well within an Openshift
   or Jenkins pipeline.
- 
+
+## Prerequisites
+1. Start the application under test
+2. Make sure you have Java installed (Min. Version 1.8)
+3. Make sure you have Gradle 4.2.1 installed: `choco install gradle --version=4.2.1`
```
Comment (jujaga) [inline tests/functional-tests/readme.md:16]: Would be good to note that the choco command is windows specific for non-windows audiences.

File: tests/openshift/templates/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                        serviceAccount: 'jenkins',
+                        cloud: 'openshift',
+                        containers: [
+                            containerTemplate(
+                                name: 'jnlp',
+                                image: 'docker-registry.default.svc:5000/<PROJECT>/bddstack:latest',
+                                resourceRequestCpu: '800m',
+                                resourceLimitCpu: '800m',
+                                resourceRequestMemory: '4Gi',
+                                resourceLimitMemory: '4Gi',
```
Comment (jujaga) [inline tests/openshift/templates/README.md:36]: We should probably review these resource limits at some point in the future, even if this is just a recommendation example.

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-vmvfjv-tools.pathfinder.gov.bc.ca/dashboard?id=common-forms-toolkit-pr-123

**Pull request merged**